centos_ci.sh

```diff
@@ -292,14 +292,42 @@ function perform_release() {
   curl http://minibot.19cf262c.svc.dockerapp.io:9009/hubot/centosci -H "Content-Type: application/json" -d '{"payload":{"status":"success","message":'"\"$MESSAGE\""',"url":'"\"$URL\""'}}' || true
 }
 
+function perform_docs_publish() {
+  # REPO and BRANCH variables are populated via https://ci.centos.org/job/minishift-docs
+  REPO_OWNER=$(echo $REPO | cut -d"/" -f4)
+  cd gopath/src/github.com/minishift/minishift
+
+  # Get changes from remote repo/branch
+  if [[ "$REPO_OWNER" != "minishift" ]]; then
+    git remote add custom-remote $REPO
+    git fetch custom-remote
+    git checkout -b remote-docs-branch --track custom-remote/$BRANCH
+  else
+    git checkout -b docs-branch --track origin/$BRANCH
+  fi
+
+  git log -n 1 # Check last commit
```

Comment:
"Check last commit" -> "Display last commit in log as reference"

You are not checking anything here